shared/constants.js

```diff
@@ -106,7 +110,7 @@ module.exports = {
             purpose: 'a hybrid mobile application',
             dir: 'hybrid',
             platforms: ['ios', 'android'],
-            toolNames: ['git', 'node', 'npm', 'cordova'],
+            toolNames: ['git', 'node', 'npm', 'cordova', 'sfdx'],
```

Comment:
NB: Only lwc templates requires sfdx - but required tools is per cli.
Also we could have other hybrid templates with a server directory in the future.